docs/labwc-actions.5.scd

```diff
@@ -80,6 +80,9 @@ Actions are used in menus and keyboard/mouse bindings.
 
 	*height* The height to resize the window to in pixels.
 
+*<action name="WarpCursorToFocused" />*
```

Comment:
`CenterCursor` seems like a brief but more meaningful name, and could also be extended to center on the currently active output if no window is focused.